src/utils/api.ts

```diff
@@ -21,99 +21,102 @@ const handleResponse = (request: Promise<AxiosResponse<any>>) =>
     return res.data
   })
 
+/* Not used yet
 const fileToFormData = (photoData: string | Blob | undefined) => {
   if (photoData !== undefined) {
     const formData = new FormData()
     return formData.append('photo_data', photoData)
   }
   return null
 }
+*/
 
 export const getClusters = (
-  params: paths['/clusters.json']['get']['parameters']['query'],
+  params: paths['/clusters']['get']['parameters']['query'],
```

Comment:
should we remove these last 3 property accessors in favor a `.` operator?

Reply:
I believe you can't use dot notation here since paths is a TS interface.